base/src/middleware/http_errors.mustache

```diff
@@ -0,0 +1,18 @@
+const HttpError = require('standard-http-error');
+const {
+  thinky: {
+    Errors: { DocumentNotFound, ValidationError }
+  }
+} = require('../../config');
+
+module.exports = (err, req, res, next) => {
+  if (err instanceof DocumentNotFound) {
+    res.status(404).send({ error: 'not found' });
```

Comment:
they could, but i don't really see the point. now all the error responses are handled in one place. and you not gonna change this, like probably ever